experiments/epopsound.py

```diff
@@ -0,0 +1,86 @@
+"""
+Copyright SuperDARN Canada 2020
```

Comment:
Maybe you could add prerequisites here in the docstring, or below in the class docstring.
For example, this depends upon a [radar].epop.passes file with a specific format